sdk/core/core-http/src/policies/ndJsonPolicy.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// BaseRequestPolicy has a protected constructor.
+/* eslint-disable @typescript-eslint/no-useless-constructor */
+
+import {
+  BaseRequestPolicy,
+  RequestPolicy,
+  RequestPolicyOptions,
+  RequestPolicyFactory
+} from "./requestPolicy";
+import { WebResourceLike } from "../webResource";
+import { HttpOperationResponse } from "../httpOperationResponse";
+
+export function ndJsonPolicy(): RequestPolicyFactory {
+  return {
+    create: (nextPolicy: RequestPolicy, options: RequestPolicyOptions) => {
+      return new NdJsonPolicy(nextPolicy, options);
+    }
+  };
+}
+
+/**
+ * NdJsonPolicy that formats a JSON array as newline-delimited JSON
+ */
+class NdJsonPolicy extends BaseRequestPolicy {
+  /**
+   * Creates an instance of KeepAlivePolicy.
+   *
+   * @param nextPolicy
+   * @param options
+   */
+  constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions) {
+    super(nextPolicy, options);
+  }
+
+  /**
+   * Sends a request.
+   *
+   * @param request
+   */
+  public async sendRequest(request: WebResourceLike): Promise<HttpOperationResponse> {
+    // There currently isn't a good way to bypass the serializer
+    if (typeof request.body === "string" && request.body.startsWith("[")) {
+      const body = JSON.parse(request.body);
```

Comment:
do we need to worry about `request.body` being non-string type?

Reply:
ok, didn't see the updated change. discard pls

Reply:
I don't believe this will cause any trouble for the way monitor uses this API today, since the mapper will serialize it to JSON when calling `track()`. We could always extend this to support streams if the API changes later.